ml/backend/ggml/ggml/src/ggml-cpu/cpu.go

```diff
@@ -1,12 +1,10 @@
 package cpu
 
-// #cgo CFLAGS: -Wno-implicit-function-declaration
+// #cgo CFLAGS: -O3 -Wno-implicit-function-declaration
```

Comment:
Do we need `CXXFLAGS` for the cpu parts of `ggml-cpu`?

Reply:
No I checked both flags and only `CFLAGS` has any effect